Tests/SwiftGRPCNIOTests/NIOServerWebTests.swift

```diff
@@ -42,24 +42,30 @@ class NIOServerWebTests: NIOBasicEchoTestCase {
     data.insert(UInt8(0), at: 0)
     return data
   }
-
-  private func gRPCWebOKTrailers() -> Data {
-    var data = "grpc-status: 0\r\ngrpc-message: OK".data(using: .utf8)!
+  
+  private func gRPCWebTrailers(status: Int, message: String) -> Data {
+    var data = "grpc-status: \(status)\r\ngrpc-message: \(message)".data(using: .utf8)!
     // Add the gRPC prefix with the compression byte and the 4 length bytes.
     for i in 0..<4 {
       data.insert(UInt8((data.count >> (i * 8)) & 0xFF), at: 0)
     }
     data.insert(UInt8(0x80), at: 0)
     return data
   }
+  
+  private func gRPCWebOKTrailers() -> Data {
+    return gRPCWebTrailers(status: 0, message: "OK")
```

Comment:
Curious, why make this 1-line function instead of inlining the other `gRPCWebOKTrailers` function?

Reply:
Not sure what you mean with that, but I changed this to `fun gRPCWebTrailers(status: Int = 0, message: String = "OK")` now.

Reply:
Sorry, typo. I was basically wondering why `gRPCWebTrailers(status: Int, message: String)` is a separate function since this is the only thing that calls it. This is fine though since `gRPCWebOKTrailers` provides clarity as to the intent

Reply:
FYI, it's actually used with a different argument in `testUnaryWithoutRequestMessage`.